report/sections/theory-and-analysis.tex

```diff
@@ -0,0 +1,6 @@
+\section{Theory and analysis}\label{sec:t-and-a}
+Unless specifically designed to be otherwise, the largest frequency component of an image is its DC component.
+The DCT process typically yields a DC value over 1000. Altering this value by $\pm 1$ results in almost no change to image quality.
+This is the core principle at the heart of many compression algorithms.\citeneeded{}
```

Comment:
We could expand this a bit by adding something along the lines of: Our watermark embedding technology relies on this DCT property for causing minimal damage to image quality and thus remaining invisible to the human eye.

You're right that this section seems sparse compared to the others. I think you have covered the major points on the theory, so there should be nothing to worry about.